openapi/bitbucket/client.bal

```diff
@@ -37,6 +39,41 @@ public client class Client {
         http:Client httpEp = check new (serviceUrl, { auth: clientConfig.authConfig, secureSocket: secureSocketConfig });
         self.clientEp = httpEp;
     }
+    # Returns a list of all repositories
+    #
+    # + workspace - This can either be the workspace ID (slug) or the workspace UUID
+    # + role - Filters the result based on the authenticated user's role on each repository. member-returns repositories to which the user has explicit read access. contributor- returns repositories to which the user has explicit write access. admin- returns repositories to which the user has explicit administrator access. owner-returns all repositories owned by the current user
+    # + query - Query string to narrow down the response as  https://developer.atlassian.com/bitbucket/api/2/reference/meta/filtering
+    # + sort - Field by which the results should be sorted as https://developer.atlassian.com/bitbucket/api/2/reference/meta/filtering
+    # + return - The repositories owned by the specified account.
+    @display {label: "List Workspaces"}
+    remote isolated function listWorkspaces(@display {label: "Workspace ID (slug) or UUID"} string workspace, 
+                                            @display {label: "User Role"} string? role = (), 
+                                            @display {label: "Filtering Query"} string? query = (), 
+                                            @display {label: "Sorting Query"} string? sort = ()) returns PaginatedRepositories|error {
+        string  path = string `/repositories/${workspace}`;
+        map<anydata> queryParam = {"role": role, "q": query, "sort": sort};
+        path = path + getPathForQueryParam(queryParam);
```

Comment:
```suggestion
        path = path + check getPathForQueryParam(queryParam);
```

Reply:
Shall we check other places using `getPathForQueryParam(queryParam)` and change it with `check` ? :)

Reply:
@lnash94 do we need to fix it manually? Haven't we patched the alpha5 tool?